src/PKSim.UI.Starter/ApplicationStartup.cs

```diff
@@ -54,24 +74,82 @@ private static IContainer initializeForStartup(IShell shell)
          container.Register<IApplicationController, ApplicationController>(LifeStyle.Singleton);
          container.Register<ICoreWorkspace, OSPSuite.Core.IWorkspace, IWorkspace, Workspace>(LifeStyle.Singleton);
 
-         // registerImplementationsFromCore(container);
          registerMinimalImplementations(container);
 
 
          using (container.OptimizeDependencyResolution())
          {
+            container.RegisterImplementationOf(shell as BaseShell);
             container.RegisterImplementationOf(shell);
+            container.RegisterImplementationOf(UserLookAndFeel.Default);
             container.AddRegister(x => x.FromType<CoreRegister>());
             container.AddRegister(x => x.FromType<InfrastructureRegister>());
             container.Register<ICreateExpressionProfilePresenter, CreateExpressionProfilePresenter>();
             container.Register<ICreateExpressionProfileView, CreateExpressionProfileView>();
+            container.Register<ICreateIndividualPresenter, CreateIndividualPresenterForMoBi>();
+            container.Register<ICreateIndividualView, CreateIndividualView>();
             container.Register<IExpressionProfileToExpressionProfileDTOMapper, ExpressionProfileToExpressionProfileDTOMapper>();
```

Comment:
So this is registered no matter what. I don't understand what you meant with the comment above @rwmcintosh